package.json

```diff
@@ -18,13 +18,11 @@
     "flow:dev": "chokidar .flowconfig flow/ src/ tests/ -i flow/logs/flow.log -c 'flow status' --initial",
     "stylelint": "stylelint --syntax scss **/*.scss",
     "lint": "npm run eslint && npm run stylelint",
-    "servertest": "bin/config-check.js && ADDONS_FRONTEND_BUILD_ALL=1 npm run build && better-npm-run servertest && better-npm-run servertest:amo && better-npm-run servertest:disco && better-npm-run servertest:admin",
     "start": "npm run version-check && NODE_PATH='./:./src' node bin/server.js",
-    "test": "bin/config-check.js && better-npm-run test",
-    "unittest": "bin/config-check.js && better-npm-run unittest",
-    "unittest:dev": "bin/config-check.js && better-npm-run unittest:dev",
-    "unittest:server": "bin/config-check.js && better-npm-run unittest:server",
-    "unittest:run": "bin/config-check.js && better-npm-run unittest:run",
+    "test-ci": "bin/config-check.js && better-npm-run test-ci",
+    "test": "bin/config-check.js && jest",
+    "test-watch": "bin/config-check.js && jest --watch",
```

Comment:
Yeah, I suppose a separate command could be useful. I think we should definitely optimize for the watch case, i.e. make it the easiest one to type: `npm test` :)